prophetess/templates/ingress.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.ingress.enabled -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ template "prophetess.fullname" . }}
+  labels:
+    app: {{ template "prophetess.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+  annotations:
+{{ if .Values.ingress.annotations -}}
```

Comment:
should this if block include the `annotations:` key? I feel like if there are no annotations, this will lead to there being the key with no value which may cause an issue with manifest parsing. Not sure if that would be the case, but just my initial thought